src/server/scripts/Spells/spell_warrior.cpp

```diff
@@ -224,16 +224,19 @@ class spell_warr_jump_to_skyhold : public SpellScriptLoader
 
```

Comment:
You should rather use the CheckCast hook and return SPELL_FAILED_NOT_HERE if not in correct zone, so the player know why his spell fail